src/main/java/AddList.java

```diff
@@ -0,0 +1,56 @@
+public class AddList {
+
+    private final String[] entries = new String[100]; 
+    private final boolean[] done = new boolean[100];   
+    private static final String SEPARATOR = "\t ___________________________"; 
+    private int index = 0; 
+
+    public void addEntry(String entry) {
+        if (index < entries.length) {
+            entries[index] = entry; 
+            index++; 
```

Comment:
You could leave a new line here to separate the main logic/data processing parts and the printing part of the conditional block/code so that it looks more structured and improves readability.